config/volume.go

```diff
@@ -343,35 +347,34 @@ func (c *Client) WatchSnapshotSignal(activity chan *watch.Watch) {
 	watch.Create(w)
 }
 
-// Validate options for a volume. Should be called anytime options are
-// considered.
-func (ro *RuntimeOptions) Validate() error {
-	if ro.UseSnapshots && (ro.Snapshot.Frequency == "" || ro.Snapshot.Keep == 0) {
-		return errored.Errorf("Snapshots are configured but cannot be used due to blank settings")
-	}
-
-	return nil
-}
-
-// Validate validates a volume configuration, returning error on any issue.
+// Validate validates a volume configuration, returning error on any
```

Comment:
These checks are moved to JSON schema. 

Schema here: https://github.com/yuva29/volplugin/blob/674460acf3628f7eb45ffdd6fbdc7e260fdc141a/config/schema.go#L5-L23 

Validation here: https://github.com/yuva29/volplugin/blob/674460acf3628f7eb45ffdd6fbdc7e260fdc141a/config/validation.go#L23-L33